tests/airgapped/README.md

```diff
@@ -127,4 +131,10 @@ To test Charmed Kubeflow components in airgapped, follow the instructions in the
 * [Katib](./katib/README.md)
 * [KNative](./knative/README.md)
 * [Pipelines](./pipelines/README.md)
-* [Training Operator](./training/README.md)
\ No newline at end of file
+* [Training Operator](./training/README.md)
+
+You will also need to ensure the airgap setup script is including the file that contains the airgap
+testing images. For a new release, we'll need to create a new dedicated file with the images needed
+for airgap testing the new release.
+
+Such images files are usually located under `tests/airgapped/ckf-X.Y-testing-images.txt`.
```

Comment:
Yes, indeed it's painful that we don't have branches so folders are our only option.

I like your approach, so with my current understanding it would look like this:
```
tests/
  airgapped/
    1.8/
      pipelines/
      training/
      testing-images.txt
      deploy.sh
    1.9/
      pipelines/
...
```

Note that in this case I also moved the `scripts/airgapped/deploy-1.8.sh` under `tests/airgapped/1.8/deploy.sh`. WDYT? Or do you think it'll be better to also have release-folders under `scripts/airgapped` as well?

For me it's also OK to include the deployment logic into tests

Reply:
@kimwnasptd for this I suggest the following:
- In the tests/airgapped directory, have the common files that do not change (e.g. `utils.sh` or the `lxc subdirectory) in the root, so their path would not change (e.g. `tests/airgapped/utils.sh`)
- Create `1.8` and `1.9` directories with the same structure, and have the different versions on the respective folder.
- Since `get-all-images.py` will be updated to work for both 1.8 and 1.9, we condense `deploy-1.8.sh` and `deploy-1.9.sh` to a Python script that receives an option like `--version`. This way we do not have to create separate directories under `scripts/airgapped`.

I can create a task for reformatting the `deploy-1.8.sh` and `deploy-1.9.sh` scripts which can be done at a later date.

Reply:
@NohaIhab @mvlassis I created a dedicated issue for this https://github.com/canonical/bundle-kubeflow/issues/1054

I suggest that we continue this discussion there and focus this effort on translating the `.sh` script to `.py` and tackle the file structure discussion orthogonally